docs/marketplace/user/merchant-portal-user-guides/202106.0/my-account/my-account.md

```diff
@@ -0,0 +1,75 @@
+---
+title: My account
+description: This topic describes what you can do in the My account area of the Merchant Portal.
+template: back-office-user-guide-template
+---
+
+This topic describes how to manage merchant user account details in the Merchant Portal.
+
+## Prerequisites
+
+To start working with the merchant user’s account, go to **Merchant Portal > My account**.
```

Comment:
this is a bit confusing. as it is actually: To open *My account* page, in the top right corner of the Merchant Portal, click the user icon and then click **My account**. so how about just removing it from the prerequisites?

Reply:
good point, thank you!